app/Models/Status.php

```diff
@@ -83,4 +89,20 @@ public function getSocialTextAttribute(): string {
 
         return $postText;
     }
+
+    /**
+     * When is a status invisible?
+     * 0=public, 1=unlisted, 2=Followers, 3=Private
```

Comment:
Maybe create an Enum for this?

Reply:
Idk, really? 🤨

Reply:
It is non-blocking for me, but would be nice. We can do it later™️.